### DISCUSSION TITLE
Move out of Range - Print Cancelled- Multi Part stl

### DISCUSSION THREAD

**Discussion by gg48gg** [xblax/flashforge_ad5m_klipper_mod#223] [Q&A]
I was printing a multi part stl file.  After finishing the third part, there is a .30mm space (equal to layer height) and then another part stacked on top.  I have done this many times to make parts self supporting (for example bottom and top of a box.  Fluidd error "Move out of range" and largest coordinates were 113.191, so nothing over 220.  The print status was cancelled at 67%.  I have not yet retried the same print.  gcode and logs attached, log excerpt below timestamp around 19:00-20:00 on 20240711.  (just short of 3 hours after print start.)

```
Stats 634.3: gcodein=0  mcu: mcu_awake=0.007 mcu_task_avg=0.000025 mcu_task_stddev=0.000016 bytes_write=113771 bytes_read=203851 bytes_retransmit=9 bytes_invalid=0 send_seq=7005 receive_seq=7005 retransmit_seq=17 srtt=0.003 rttvar=0.001 rto=0.025 ready_bytes=16 stalled_bytes=0 freq=144001976 eboard: mcu_awake=0.002 mcu_task_avg=0.000017 mcu_task_stddev=0.000012 bytes_write=12725 bytes_read=50812 bytes_retransmit=9 bytes_invalid=0 send_seq=1168 receive_seq=1168 retransmit_seq=2 srtt=0.002 rttvar=0.001 rto=0.025 ready_bytes=22 stalled_bytes=0 freq=144002712 adj=144000811 heater_bed: target=60 temp=60.0 pwm=0.338 Load_Cell: temp=0.0 TVOC_Level: temp=23.0 sd_pos=6708 sysload=0.30 cputime=43.753 memavail=29244 print_time=641.306 buffer_time=2.192 print_stall=0 extruder: target=200 temp=199.8 pwm=0.246
Move out of range: 113.191 -70.413 -1.718 [43.063]
```
[sample.zip](https://github.com/user-attachments/files/16188971/sample.zip)
[logs.zip](https://github.com/user-attachments/files/16189031/logs.zip)

**Comment by gg48gg** [comment]
Retry after printer reboot, same result.
Printed successfully on Bambu Lab P1P.

**Reply by consp** [reply]
> Bambu Lab P1P

Does that one use klipper?

My guess would be it's a setting somewhere but I'm not an expert on klipper settings, @xblax do you have an idea?

**Comment by jeffalanmcclain** [comment]
I've had this happen quite a lot as well, and reslicing/printing again worked.  I also managed to catch ONE time where I just reprinted the exact same sliced upload file successfully, after a reboot.  I believe this is a memory overflow issue.?
